u-mode/src/main.rs

```diff
@@ -121,5 +122,6 @@ extern "C" fn task_main(cpuid: u64, shared_addr: u64, shared_size: u64) -> ! {
         "umode/#{}: U-mode Shared Region: {:016x} - {} bytes",
         cpuid, shared_addr, shared_size
     );
+    test_declare_pass!("umode start", cpuid);
```

Comment:
minor: I think it would be better to do this in Salus after the U-mode task calls back into Salus for its next task in `run_loop()`.

Reply:
or both.